pallets/process-validation/src/lib.rs

```diff
@@ -79,50 +100,113 @@ pub mod pallet {
 
     /// Storage map definition
     #[pallet::storage]
-    #[pallet::getter(fn processes_by_id_and_version)]
-    pub(super) type ProcessesByIdAndVersion<T: Config> = StorageDoubleMap<
+    #[pallet::getter(fn get_process)]
+    pub(super) type GetProcess<T: Config> = StorageDoubleMap<
```

Comment:
I don't particularly like the storage name here as `GetProcess` feels like it describes a function for purely "getting". I think either `Processes` or as it was before `ProcessesByIdAndVersion` would feel closer

Reply:
Yes! well spotted, just wanted to keep something simple, easier to remember, will update to Process and Version I guess because i see it as ORM/db model. Please let me know what would you prefer